client/homebrew/editor/editor.jsx

```diff
@@ -151,15 +151,21 @@ const Editor = createClass({
 
 						// definition lists
 						if(line.includes('::')){
-							const regex = /^([^\n]*?)::([^\n]*)(?:\n|$)/ym;
+							if(/^:*$/.test(line) == true){ return };
+							const regex = /^([^\n:]*?)(::[^\n]*)(?:\n|$)/ymd;  // the `d` flag, for match indices, throws an ESLint error.
```

Comment:
I *think* you can just get rid of the first colon on this line, to make this:
```regex
/^([^\n]*?)(::[^\n]*)(?:\n|$)/ymd
```
That colon was added on Jan 7th, according to Blame, and I wonder if it is a remnant from a previous syntax that was tried?

This is to address this lack of highlight:
![image](https://github.com/naturalcrit/homebrewery/assets/58999374/50e62fbd-e003-45af-99b8-0a56977ce722)

```
Term: :: definition
```

Reply:
That mostly fixed it, but I had to throw in a `:?\s?` at the end of the first capture group to catch `term:::definition` cases.